devops  alona

```diff
@@ -0,0 +1,20 @@
+from flask import Flask, request
+import subprocess
+
+app = Flask(__name__)
+
+@app.route('/system-info', methods=['GET'])
+def system_info():
+    command = request.args.get('command')
+    if not command:
+        return "No command ", 400
+
```

Comment:
It is recommended to add an input check, to verify it is command with certain rules and verifications. need to do that externally to the api and use it here